astro/src/content/quickstarts/quickstart-php-laravel-web.mdx

```diff
@@ -0,0 +1,349 @@
+
+In this file, also overwrite the values for the following keys as described here.
+
+```
+DB_HOST=mariadb
+DB_USERNAME=bn_myapp
+DB_DATABASE=bitnami_myapp
+```
+
+This enables {frontmatter.technology} to connect to the database running in the adjacent Docker container.
+
+<Aside type="note">
+  Note that {frontmatter.technology} will silently overwrite the `DB_HOST` value with `mariadb` in the file `InteractsWithDockerComposeServices.php`, so you can't change this value or the database service name in Docker. If you encounter database connection errors in your own apps running in Docker, check that this file isn't interfering with your connection strings.
+</Aside>
+
+In the file `your-application/config/services.php`, add one more item to the array being returned with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/config/services.php"}
+  lang="php"
+  tags="fusionauth" />
+
+Now the configuration files have all the settings you need. Next, you'll change your provider to be aware of FusionAuth.
+
+In the file `your-application/app/Providers/EventServiceProvider.php`, replace the `$listen` array with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/app/Providers/EventServiceProvider.php"}
+  lang="php"
+  tags="fusionauth" />
+
+Finally, update your User model to have FusionAuth fields.
+
+In the file `your-application/app/Models/User.php`, add the FusionAuth items to the `$fillable` and `$hidden` properties with the code below.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/app/Models/User.php"}
+  lang="php"
+  tags="fusionauth" />
+
+Every model in {frontmatter.technology} uses the database, so you need to update the database table to match your new model.
+
+Create a file named `your-application/database/migrations/2023_09_27_142009_add_fusionauth_fields_user_table.php` or run the command below to create it.
+
+<Aside type="note">
+  Running the command below will create the file with a slightly different name depending on the date you are running the command.
+</Aside>
+
+```shell
+docker compose exec lara_app php artisan make:migration add_fusionauth_fields_user_table
+```
+
+The result is the same whether you create the database scripts manually or with this command, so you can use your preferred method.
+
+Add the code below to the file.
+
+<RemoteCode
+  url={frontmatter.codeRoot + "/complete-application/laravel/database/migrations/2023_09_27_142009_add_fusionauth_fields_user_table.php"}
+  lang="php" />
+
+Run the command below to update your MariaDB database with the new User definition.
+
+```shell
+docker compose exec lara_app php artisan migrate
+```
+
+## Customization
+
+Now that authentication is done, the last task is to create example pages that a user can access only when logged in.
+
+### CSS And HTML
+
+Copy over some CSS and an image from the example app.
```

Comment:
Can we please do the same thing with the changebank logo, rather than pulling it from the fusionauth.io website via a `src` tag?

Reply:
Fixed in this PR https://github.com/FusionAuth/fusionauth-quickstart-php-laravel-web/pull/2